### DISCUSSION TITLE
How to access what/where the list of parent/child relationship for tabs? (a project of mine)

### DISCUSSION THREAD

**Discussion by ForDefault** [mbnuqw/sidebery#950] [Q&A]
Hey! I am working on a project to integrate with sidebery. (wont go into it unless someone wants to know lol)
### I am stuck on how to get the output or access:

1.  All parent/child tabs opened in a window (and then all windows opened). 
2. Does sidebery "know" what child tab moves from one html to another? I am asking regarding wanting to make a history of each parent/child tab 

Any direction or help on this would be awesome! thank you

**Comment by mbnuqw** [comment]
You can try to check Tab.openerTabId property using the tabs API to get tabs tree. Unfortunately, right now there is no way to get sidebery-panel/tab relation, but it's planned to add an API to get full sidebery tree info for other addons.
Issue to track: #616
